test/e2e/network/service.go

```diff
@@ -3431,3 +3434,83 @@ func validateEndpointsPorts(c clientset.Interface, namespace, serviceName string
 	}
 	return fmt.Errorf("Timed out waiting for service %s in namespace %s to expose endpoints %v (%v elapsed)", serviceName, namespace, expectedEndpoints, framework.ServiceStartTimeout)
 }
+
+// restartApiserver restarts the kube-apiserver.
+func restartApiserver(namespace string, cs clientset.Interface) error {
+	// TODO: Make it work for all providers.
```

Comment:
If we make it private here, is the TODO still valid?

Reply:
/cc @wongma7 
for this TODO

Reply:
The test itself doesn't seem cloud provider specific, so I don't think we should remove this TODO at this time.